cmake/modules/GetGitRevisionDescription.cmake

```diff
@@ -87,6 +87,111 @@ function(get_git_head_revision _refspecvar _hashvar)
             PARENT_SCOPE)
 endfunction()
 
+function(git_get_latest_tag _var)
+    if(NOT GIT_FOUND)
+        find_package(Git QUIET)
+    endif()
+
+    # We use git describe --tags `git rev-list --tags --max-count=1`
+    execute_process(COMMAND
+            "${GIT_EXECUTABLE}"
+            rev-list
+            ${ARGN}
```

Comment:
Only difference from Falco one: we pass `--tags` too from external, because in `git rev-list`, `--tags` option takes an optional filtering argument, and we need to filter for `+driver`.
See https://git-scm.com/docs/git-rev-list.